lib/add-allegation-data.js

```diff
@@ -4,57 +4,55 @@ const _ = require('lodash');
 const fieldsMap = require('./ims-hof-fields-map.json');
 const valuesMap = require('./ims-hof-values-map.json');
 
-const eform = {
-  EformFields: [],
-  AdditionalPeople: [],
-  Attachments: []
-};
-
-const addFieldToEform = (fieldName, fieldValue) => {
+const addFieldToEform = (eform, fieldName, fieldValue) => {
   if (fieldName === 'images') {
-    eform.Attachments = fieldValue;
-  } else if (fieldName === 'txpermoreallholder') {
-    eform.AdditionalPeople.push({FieldName: fieldName, FieldValue: fieldValue});
+    eform.Attachements.push({url: fieldValue});
+  } else if (fieldName === 'tbwhoaddperson') {
+    eform.AdditionalPeople = fieldValue;
```

Comment:
For attachments it needs to be `eform.Attachments = fieldValue;` here on line 9 as well as with additional people on line 11 otherwise that part of the array is also unnecessarily deep. I think it is that way on master branch right now from #106 or #119.

Reply:
basically both lines should be `eform.Thing = fieldValue;`